.github/workflows/empty-worksapce-smoke-test-on-minikube.yaml

```
@@ -0,0 +1,88 @@
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+name: Empty workspace smoke test
+on:
+  pull_request:
+    types:
+      - opened
+
+env:
+   USERSTORY: CloneGitRepoAPI
+   TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL: kubectl
+   DEPLOYMENT_TIMEOUT: 90s
+
+jobs:
+  buildImageRunner:
```

It looks like other jobs use [snake_case](https://en.wikipedia.org/wiki/Snake_case)

```suggestion
  build_image_runner:
```